src/GlassCat/GlassTypes.jl

```diff
@@ -16,13 +16,6 @@ struct GlassID
 end
 
 Base.show(io::IO, a::GlassID) = print(io, "$(string(a.type)):$(a.num)")
-type(a::GlassID) = a.type
-num(a::GlassID) = a.num
-
-ismodel(a::GlassID) = a.type === MODEL
-isMIL(a::GlassID) = a.type === MIL
-isAGF(a::GlassID) = a.type === AGF
-isother(a::GlassID) = a.type === OTHER
```

Comment:
I had a look at some Julia code style discussions online and it seems that getters and setters aren't seen as strictly necessary due to the ability to override Base.getproperty. Any objections?